app/admin/contributions.rb

```diff
@@ -26,7 +26,10 @@
   end
   batch_action :approve, if: proc { params[:scope] != "recusado" && params[:scope] != "aprovado" } do |ids|
     batch_action_collection.find(ids).each do |contribution|
-      contribution.approve!(current_user.id) if contribution.received?
+      if contribution.received?
+        contribution.approve!(current_user.id)
+        NotificationMailer.notify_fill_contribution_description(contribution).deliver_now
```

Comment:
this should be sent when is received, trigger the email right after creating the contribution

Reply:
Done